### ISSUE TITLE
Question - Can SendBox's private borderAndBoxShadowStyle be changed without customizing the source code?

### ISSUE THREAD

**Issue by managedcoder** [Azure/communication-ui-library#3086]
I've found the style properties of the SendBox that I want to change, and they live in SendBox's borderAndBoxShadowStyle, but after looking at the code, I am wondering if there's some way to change those styles in my app instead of modifying the source and creating a customized version of UI Library.  I just want to make sure there isn't some provision for changing styles like this that I'm not aware of before I go and customize source code.  Seems like I am going through a lot of effort to change the border style of the SendBox so I wanted to double check before I go charging off with changes like this.

**Comment by jpeng-ms** [comment]
Hi Russ,

Thanks for reaching out to us. This is John and I will be helping you this week. Please allow us to look into this and we would get back to you shortly.

John

**Comment by jpeng-ms** [comment]
Hi Russ,

Thanks for reaching out to us. This is John and I will be helping you this week. Please allow us to look into this and we would get back to you shortly.

John

**Comment by jpeng-ms** [comment]
Hi Russ,

Thanks for your patience. I have checked with our team and this is an oversight from us. There should've had a field `styles` field that takes in `root` just like the other components we have where you can define or overwrite any CSS based on your preference:

```js
  const personaOptions: IPersona = {
    hidePersonaDetails: true,
    size: PersonaSize.size24,
    text: person.displayName,
    showOverflowTooltip: false,
    styles: {
      root: {
        margin: '0.25rem'
      }
    }
  };
```
We have created a backlog item to track this as an enhancement. In the meanwhile, I was wondering if you could implement some walk-around so you can be unblocked? Specifically, I was wondering if you could wrap the send box component in a parent `<div>` that has styles being marked as `!important`, similar to this:

```js
export const SendBoxExample: () => JSX.Element = () => (
  <FluentThemeProvider>
    <div style={{ margin: '0 auto !important' }}>
      <SendBox
        onSendMessage={async () => { ... }}
        onTyping={async () => { ... }}
      />
    </div>
  </FluentThemeProvider>
);

```

Please be mindful that this walk-around is only supposed to be used temporality while we are making enhancements to the `sendbox` component. Please note that it makes your code inflexible and might break other CSS if you have multiple class defined in the same scope. In addition, `borderAndBoxShadowStyle` might be changed in the future and this walk-around might be broken.

Please let me know if you have any other questions.

John

**Comment by jpeng-ms** [comment]
ADO #3277414

**Comment by JoshuaLai** [comment]
@managedcoder We have a provided a quick code sample that you can try out, and created a task in our backlog to track this as a bug on our side. Let me know if there is any more clarification you may need for this.

**Comment by JoshuaLai** [comment]
We have this as a work item tracked in our back log but don't have a time line yet for completion. But we also suggested some code sample as well. I will be closing this issue. If there are any follow up questions please feel free to open a new issue, or leave a comment here, and we can reopen this issue if needed.
